src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -4294,6 +4294,12 @@ private OneOrMany<Symbol> GetMethodGroupSemanticSymbols(
                         // we want to get the symbol that overload resolution chose for M, not the whole method group M.
                         var conversion = (BoundConversion)boundNodeForSyntacticParent;
 
+                        if (conversion.ConversionKind is not ConversionKind.MethodGroup && conversion.Operand is BoundConversion nestedConversion)
```

Comment:
>nestedConversion [](http://example.com/codeflow?start=129&length=16)

It feels like we should also check how this node related to the conversion syntactically